action.yml

```diff
@@ -153,4 +153,7 @@ runs:
           diff -B west.yml tmp.yml | patch west.yml - || true
           git add west.yml
           git commit --amend --no-edit
+          git fetch origin
+          git merge-tree --write-tree --name-only origin/main auto-manifest-${{ github.event.repository.name }}-${{ github.event.pull_request.number }}
+          if (( $? == 1)) ; then git rebase origin/main -X theirs ; fi
```

Comment:
We could also add logic to close the PR if we detect that the new SHA is older than what is being used in the parent repo

Reply:
Since this is run on `github.event.pull_request.merged` The window for someone else to merge something newer should be quite small so we can keep it as is for now.